src/pset/map/output.rs

```diff
@@ -75,12 +89,16 @@ pub struct Output {
     /// corresponding master key fingerprints and derivation paths.
     #[cfg_attr(feature = "serde", serde(with = "::serde_utils::btreemap_as_seq"))]
     pub bip32_derivation: BTreeMap<PublicKey, KeySource>,
-    /// (PSET2) The amount of the output
-    pub amount: confidential::Value,
+    /// (PSET2) The explicit amount of the output
+    pub amount: Option<u64>,
+    /// (PSET2) The out amount commitment
+    pub amount_comm: Option<secp256k1_zkp::PedersenCommitment>,
```

Comment:
In 9840052067bd91bf09cf65001f307f95a47abca1:

I think you should drop `PSET2` from all of these. There are no other versions of PSET.